frontend/packages/kubevirt-plugin/src/components/vms/vm-resource.tsx

```diff
@@ -93,6 +94,8 @@ export const VMDetailsList: React.FC<VMResourceListProps> = ({ vm, vmi, pods, mi
           DASH
         )}
       </dd>
+      <dt>CD</dt>
+      <dd id={prefixedID(id, 'cd-rom')}>{cd.length > 0 ? cd[0].name : 'No Disk'}</dd>
```

Comment:
we only first one and not all of them ?

Reply:
OK, i see that the design shows only one CD-ROM but we can have more @matthewcarleton

Reply:
So there could be multiple Cds? ok, should we display them in some kind of order? @rawagner

Reply:
according to docs https://kubevirt.io/user-guide/docs/latest/creating-virtual-machines/disks-and-volumes.html#cdrom any disk with 'cdrom' attribute is CDRom and it does not mention we can have only one. CDRom can also have boot order assigned (as any disk) so maybe we want to order them by boot order, then just by position in yaml ?